test/support/meetups/sample_api_responses.rb

```diff
@@ -0,0 +1,317 @@
+    "description"=>
+     "<p>Operation Code is an initiative to introduce veterans to the software development community. In this group, we will work together to design a website for aggregating veteran resources.</p> <p>No experience is necessary to join. In fact, we encourage those who want to learn how to code to join us! If you are a developer and would like to provide mentorship, we would greatly appreciate your participation, as well.</p> <p>In this meetup, we will discuss the progress that we've made and the next items on our list to complete. When the team is clear about the objectives, we will break into teams for pair programming and continue to build our website.</p> <p>Feel free to contact Larry Burris if you have any questions.</p> ",
+    "visibility"=>"public"},
+   {"created"=>1506361286000,
+    "duration"=>7200000,
+    "id"=>"243655170",
+    "name"=>"Come Learn To Code!",
+    "status"=>"upcoming",
+    "time"=>1513899000000,
+    "updated"=>1506361286000,
+    "utc_offset"=>-18000000,
+    "waitlist_count"=>0,
+    "yes_rsvp_count"=>2,
+    "venue"=>
+     {"id"=>24628300,
+      "name"=>"1701",
+      "lat"=>36.844764709472656,
+      "lon"=>-75.97899627685547,
+      "repinned"=>false,
+      "address_1"=>"1701 Baltic Avenue",
+      "city"=>"Virginia Beach",
+      "country"=>"us",
+      "localized_country_name"=>"USA",
+      "zip"=>"",
+      "state"=>"VA"},
+    "group"=>
+     {"created"=>1483543375000,
+      "name"=>"Operation Code: Hampton Roads",
+      "id"=>21767819,
+      "join_mode"=>"open",
+      "lat"=>36.900001525878906,
+      "lon"=>-76.20999908447266,
+      "urlname"=>"Operation-Code-Hampton-Roads",
+      "who"=>"coders",
+      "localized_location"=>"Norfolk, VA",
+      "region"=>"en_US"},
+    "link"=>"https://www.meetup.com/Operation-Code-Hampton-Roads/events/243655170/",
+    "description"=>
+     "<p>Operation Code is an initiative to introduce veterans to the software development community. In this group, we will work together to design a website for aggregating veteran resources.</p> <p>No experience is necessary to join. In fact, we encourage those who want to learn how to code to join us! If you are a developer and would like to provide mentorship, we would greatly appreciate your participation, as well.</p> <p>In this meetup, we will discuss the progress that we've made and the next items on our list to complete. When the team is clear about the objectives, we will break into teams for pair programming and continue to build our website.</p> <p>Feel free to contact Larry Burris if you have any questions.</p> ",
+    "visibility"=>"public"},
+   {"created"=>1506361383000,
+    "duration"=>7200000,
+    "id"=>"243655210",
+    "name"=>"Come learn to code!",
+    "status"=>"upcoming",
+    "time"=>1516318200000,
+    "updated"=>1506361383000,
+    "utc_offset"=>-18000000,
+    "waitlist_count"=>0,
+    "yes_rsvp_count"=>2,
+    # "venue"=>
```

Comment:
I commented this out so I can test that events without venue information do not save.